.github/pr-title-checker-config.json

```
@@ -0,0 +1,13 @@
+{
+    "LABEL": {
+        "name": ""
+    },
+    "CHECKS": {
+        "regexp": "gh-(\\d+): "
```

Does this need to be a white space character?
```suggestion
        "regexp": "gh-(\\d+):\\s"
```